src/fides/api/schemas/privacy_experience.py

```diff
@@ -16,20 +17,46 @@ class ExperienceConfigSchema(FidesSchema):
     """
     Base for ExperienceConfig API objects.  Here all fields are optional since
     Pydantic allows subclasses to be more strict but not less strict
-    """
 
-    acknowledgement_button_label: Optional[SafeStr]
-    banner_title: Optional[SafeStr]
-    banner_description: Optional[SafeStr]
-    component: Optional[ComponentType]
-    component_title: Optional[SafeStr]
-    component_description: Optional[SafeStr]
-    confirmation_button_label: Optional[SafeStr]
-    delivery_mechanism: Optional[DeliveryMechanism]
-    disabled: Optional[bool] = False
-    is_default: Optional[bool] = False
-    link_label: Optional[SafeStr]
-    reject_button_label: Optional[SafeStr]
+    Note component is intentionally not included here.  When creating component, you can add it, but when updating,
+    the component cannot be updated.
+    """
+
+    accept_button_label: Optional[SafeStr] = Field(
+        description="Overlay 'Accept button displayed on the Banner and Privacy Preferences' or Privacy Center 'Confirmation button label'"
```

Comment:
I've tried to mark here how the fields are shared between the Overlay Experience Config and the Privacy Center Experience Config.

Reply:
this was very helpful, thank you!